### ISSUE TITLE
Missing model definitions for imdbwiki_googlenet for both age and gender networks.

### ISSUE THREAD

**Issue by hosaka** [sebastian-lapuschkin/understanding-age-gender-deep-learning-models#2]
Hello Sebastian, thank you for your fantastic work first of all! I am trying to re-train some of the networks with my own data. In your weights archive I found and downloaded the age/gender_imdbwiki_googlenet models. However in the repos model_definitions only vgg16 network is present with imdbwiki configuration.

**Comment by hosaka** [comment]
@sebastian-lapuschkin would you be so kind to share the prototxt files for imdbwiki_googlenet configurations? For both age and gender networks. Thank you!

**Comment by sebastian-lapuschkin** [comment]
Hi @hosaka,

Since there have not been any pre-trained weights for the bvlc_googlenet architecture and the imdbwiki dataset at the time of publication, there are no corresponding models pretrained on imagenet and imdbwiki, except for the vgg16 model.

Focus of the paper has been an attempt to recreate some models from literature while investigating some of the major (fluctuating) points of change between their configurations, and manually pretraining on imdbwiki has proven to be difficult and very time consuming for reliably preparing the required number of models. In addition, a manual pretraining would have constituted another deviation from reference models; imdbwiki requires a lot of bug fixing and introduction of random sampling in order be usable properly.
I therefore decided to not include those models into the evaluation.

Whatever you can find in the files is all there is.

**Comment by sebastian-lapuschkin** [comment]
I will add the solver*.prototxt files, which contain e.g. learning rates and further training parameters, though.
That should get you somewhat closer to recreating the paper's results.

Give me some minutes.

**Comment by sebastian-lapuschkin** [comment]
I have added all the solver* files corresponding to the models.
Note though, that the number of training iterations and iterations between model snapshots are currently inadequately set:

After training the models "to completion", I wanted to have a closer look at training progress in the early iterations. Thus, I have increased snapshot frequency and limited the max number of iterations accordingly.

The number of iterations of training for the results reported in the paper -- depending on model architecture, pretraining and input data variant --  can be inferred from the file

`evaluation_scripts/score_models.py`, lines 166-187

I hope that helps. Feel free to close the issue if your questions have been answered.
